### DISCUSSION TITLE
Issue in Running Conductor Locally - RedisDynoQueue

### DISCUSSION THREAD

**Discussion by ropereraLK** [Netflix/conductor#3490] [General]
I have cloned the conductor from GitHub and followed the instruction given in the following documentation (https://conductor.netflix.com/gettingstarted/server.html) to set up conductor locally.
 Steps followed: 
       $ git clone https://github.com/Netflix/conductor.git
       $ cd conductor
       conductor $ cd server
       server $ ../gradlew bootRun

But after 99% execution completes the command prompt gives the following warn log and keeps showing the same error log in an endless loop.

**"595706 [scheduled-task-pool-1] WARN  com.netflix.dyno.queues.redis.RedisDynoQueue [] - cannot add 47cc7ed3-c060-4864-b99b-9a1d7c30f3df to the unack shard .test.UNACK._deciderQueue.c"**

Java Version used: Java 11
OS: Windows

Can someone help me on this.?

**Comment by manan164** [comment]
Hi @ropereraLK , this is the warning logs.  We can ignore that. Alternatively, you can use [orkes-queues](https://github.com/orkes-io/orkes-queues) in place of dyno queues. Let me know if this works, or we can chat [here](https://join.slack.com/t/orkes-conductor/shared_invite/zt-xyxqyseb-YZ3hwwAgHJH97bsrYRnSZg) for more realtime collaboration.

**Reply by ropereraLK** [reply]
> Hi @ropereraLK , this is the warning logs. We can ignore that. Alternatively, you can use [orkes-queues](https://github.com/orkes-io/orkes-queues) in place of dyno queues. Let me know if this works, or we can chat [here](https://join.slack.com/t/orkes-conductor/shared_invite/zt-xyxqyseb-YZ3hwwAgHJH97bsrYRnSZg) for more realtime collaboration.

Hi @manan164 Thank you. I will try and get back to you.
